app/src/LeadsPage.tsx

```diff
@@ -0,0 +1,146 @@
+import axios from "axios";
+import "./LeadsPage.scss";
+import { useState, useEffect, PropsWithChildren } from "react";
+
+const server = axios.create({
+  baseURL: "http://localhost:5173",
+});
+
+interface Lead {
+  id: string;
+  createdAt: string;
+  priority: "Low" | "Medium" | "High";
+  stage:
+    | "New"
+    | "Acknowledged"
+    | "Negotiation"
+    | "Contract Sent"
+    | "Customer"
+    | "Closed";
+  owner?: string;
+  firstName: string;
+  lastName: string;
+  phoneNumber: string;
+  email: string;
+  jobTitle: string;
+  companyName: string;
+  sector: string;
+  employeeCount?: number;
+  address?: string;
```

Comment:
there is alreday a file that defined a "lead" type...

Reply:
lilach and me have to decide on a way to pull out the leads into a nested object.